providers/capa/cilium-eni-mode/capa_test.go

```diff
@@ -0,0 +1,53 @@
+package cilium_eni_mode
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/giantswarm/cluster-test-suites/internal/common"
+	"github.com/giantswarm/cluster-test-suites/internal/state"
+)
+
+var _ = Describe("Cilium ENI mode tests", Ordered, func() {
```

Comment:
Why is this test suite set as an ordered suite?

Reply:
I thought I need it to run

```
It("assigns IP addresses from secondary VPC CIDR to pods", func() {
```

last, i.e. right before the suite destroys the WC. If I don't do this, wouldn't it be possible that these assertions run before `common.Run` i.e. when the WC isn't ready yet? I could be wrong though 😉!

Reply:
Setting it to ordered does mean that if any of the test cases fail (e.g. a flake) it doesn't attempt to run the rest of the test cases that come after it. Is that something you want? Normally (with the standard test suites) we want to attempt to run all test cases and then report back on all the cases that failed to save needing to re-run to discover more.

Reply:
Hmm, I understood from [ginkgo documentation](https://onsi.github.io/ginkgo/#ordered-containers) that `Ordered` ensures that `It` blocks run in order, and thought we need that here since the common stuff (incl. cluster creation and health check) must definitely come first before I can check some other details about the running workload cluster. Are you sure about the behavior you described above?

Reply:
We don't run tests in parallel so the order is already confirmed. I'm sure about the behaviour and you can see it happen in your previous test run here: https://tekton.ci.giantswarm.io/#/namespaces/tekton-pipelines/pipelineruns/pr-cluster-test-suites-287-cluster-test-suitesz8csx?pipelineTask=run-tests&step=run-tests&taskRunName=pr-cluster-test-suites-287-cluster-test-suitesz8csx-run-tests-0

Reply:
Okay, I'm trying one test run without the `Ordered` flag

Reply:
Just to be clear, I'm ok with it having `Ordered` if you think it adds benefit. Just as long as you're aware of the impact